### ISSUE TITLE
Unclear instructions in Dimentionality reduction chapter in Data Science and Machine Learning.

### ISSUE THREAD

**Issue by eden500** [Galalon/CV_training#4]
The last two question about calculating the precision are unclear. It should be noted that there is a directory with pairs of images of the same persons and those are the positives.

**Comment by shalev-habany** [comment]
I agree with Eden, The metric we have asked to define is a metric that takes image and its reconstraction and returns us true or false, true means its the reconstruction of the image and false means its not the reconstruction of the image.
However in order to calculate meaningful precision we need to have negative gt (we have only positive gt) so we can take the other images and label them as false for the other's image reconstraction then fp will be that the matrix says true to (reconstruction of image 0, image 1).
If you are not understanding me the idea is similar to simclr (just without augmentations)
